lib/screens/editor/tags.dart

```diff
@@ -39,7 +39,7 @@ class _TagEditorPageState extends State<TagEditorPage> {
 
   String _getUrl() => 'https://$kOsmAuth2Endpoint/${widget.amenity.id.fullRef}';
 
-  String _getHistoryUrl() => '${_getUrl()}/history';
+  Uri _getHistoryUrl() => Uri.parse('${_getUrl()}/history');
```

Comment:
Maybe move this function to `versions.dart` as well? Since launching was also moved there.

Reply:
Fixed here 45981e2